functions/v2/autoLabelInstance/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "nodejs-docs-samples-functions-v2",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "scripts": {
+    "test": "mocha test/index.test.js",
+    "e2e-test": "mocha --timeout 60s test/*.test.js"
```

```suggestion
    "e2e-test": "mocha --timeout 60s test/system.test.js"
```
so unit tests are reran